bin/start_be.sh

```diff
@@ -20,6 +20,13 @@ set -eo pipefail
 
 curdir="$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
 
+if [ $(id -u) -eq 0 ]; then # current user has root privileges
+    # system will no longer throw bad_alloc, memory alloc are always accepted,
```

Comment:
maybe we should not do that. it's better to print a warning? or add the message to be.out?
but i am not sure.

Reply:
You are right, should not directly modify sys conf
Changed to print log